src/modules/roc_pipeline/receiver_session.h

```diff
@@ -52,6 +52,7 @@ class ReceiverSession : public core::RefCnt<ReceiverSession>, public core::ListN
                     const size_t out_sample_rate,
                     const packet::Address& src_address,
                     const rtp::FormatMap& format_map,
+                    const unsigned int payload_type,
```

Comment:
Nitpick: I'd move this field near `ReceiverConfig config_`, which contains other related session parameters, like sample rate and channel mask.

Reply:
faa175e0022dcfee68511bbe6a36e018b3428b86